src/python/WMCore/ReqMgr/Web/ReqMgrService.py

```diff
@@ -211,6 +211,27 @@ def __init__(self, app, config, mount):
         identifier = cdict.get("log_reporter", "reqmgr2")
         self.logdb = LogDB(centralurl, identifier)
 
+        # local team cache which will request data from wmstats
+        self.wmstatsurl = config.reqmgr.wmstats_url
```

Comment:
How about keeping the same style as in the previous lines, e.g.:
self.wmstatsurl = cdict.get('wmstats_url', '')

I'd also get rid of the if in the line below, we simply assume it will be in the configuration file.
Otherwise, we better than check whether all the previous URLs are also available in the config file.